topo_processor/metadata/metadata_validators/metadata_validator_imagery_historic.py

```diff
@@ -0,0 +1,21 @@
+from topo_processor.stac.collection import Collection
+from topo_processor.stac.data_type import DataType
+from topo_processor.stac.item import Item
+from topo_processor.util.tiff import is_tiff
+
+from .metadata_validator import MetadataValidator
+
+
+class MetadataValidatorImageryHistoric(MetadataValidator):
+    name = "validator.imagery.historic"
+
+    def is_applicable(self, item: Item) -> bool:
+        if item.data_type != DataType.ImageryHistoric:
+            return False
+        if not is_tiff(item.source_file):
+            return False
+        return True
+
+    async def validate_metadata(self, item: Item) -> None:
+        item.is_valid = True
+        item.output_dir = item.stac_item.properties["linz:survey"]
```

Comment:
this would be the prime spot to add the collection.json reference

Reply:
that will mean we will have to pass collection to this function and change the whole api